frontend/src/libs/components/popover/popover.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  content: React.ReactNode;
+  trigger: React.ReactNode;
+};
+
+const Popover: React.FC<Properties> = ({
+  content,
+  trigger,
+}: Properties): JSX.Element => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const handleClick = useCallback((): void => {
+    setIsOpen(!isOpen);
```

Comment:
Maybe better `setIsOpen(prevState => !prevState)` (more info https://stackoverflow.com/questions/61054275/usestate-with-boolean-value-in-react)

Reply:
but i'm using useCallback hook, it's not fixing a problem?

Reply:
can be both